feat: add json_keys UDF (#8603)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/scalar-functions.md

    `CREATE STREAM LOGS (LOG STRUCT<CLOUD STRING, APP STRING, INSTANCE INT>, ...) WITH (VALUE_FORMAT='JSON', ...)`

### `JSON_KEYS`

Since: 0.25.0

```sql
json_keys(json_string) -> Array<String>
```

Given a string, parses it as a JSON object and returns a ksqlDB array of strings representing the

> top-level keys. Returns `NULL` if the string can't be interpreted as a JSON object, i.e., it is

Reviewer: ```suggestion
top-level keys. Returns `NULL` if the string can't be interpreted as a JSON object, for example, when it is
```
